src/main/java/org/folio/rest/impl/CalendarAPI.java

```diff
@@ -65,19 +68,16 @@
 import org.joda.time.DateTime;
 
 import io.vertx.core.AsyncResult;
-import io.vertx.core.CompositeFuture;
 import io.vertx.core.Context;
 import io.vertx.core.Future;
 import io.vertx.core.Handler;
 import io.vertx.core.Promise;
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
 
 
 @SuppressWarnings({"squid:S1854", "squid:S1481"})
 public class CalendarAPI implements Calendar {
 
-  private static final Logger logger = LoggerFactory.getLogger(CalendarAPI.class);
+  private static final Logger logger = LogManager.getLogger(CalendarAPI.class);
```

Comment:
Should we try initializing the logger with no arguments?

> using getLogger() without argument requires Multi-Release for stack walking

We do have Multi-Release enabled in POM.
https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-320